scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -28,14 +28,14 @@
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "",
+    "@fluentui/react-conformance": "",
+    "@fluentui/scripts": "",
     "@types/enzyme": "",
     "@types/enzyme-adapter-react-16": "",
     "@types/jest": "",
     "@types/react": "",
     "@types/react-dom": "",
     "@types/react-test-renderer": "",
-    "@types/webpack-env": "",
-    "@fluentui/scripts": "",
     "enzyme": "",
```

Comment:
Other dep fixes: `react-conformance` was missing, and `webpack-env` isn't actually needed